.gitignore

```diff
@@ -98,6 +101,7 @@ background.tiff*
 
 # Qt Creator
 Makefile.am.user
+build-bitcoin-qt*
```

Comment:
What or why would this file be created?

Reply:
Folders with this prefix are created by Qt creator.